openshift/aws-account-operator
# Put aws accounts in organization ou

Ticket: https://issues.redhat.com/browse/OSD-1603

For billing and reporting purposes put all AWS Accounts into an OU  in the AWS Organization where the OU represents the customer's organization / account.  This will be based on AccountClaim's legalEntity.id field. 

Done:
- When Account status is updated with a legalEntity.id (and was not set prior):
- Create OU in AWS Organization for the .status.legalEntity.id if it doesn't already exist.
- Move the AWS Account to the OU representing the .status.legalEntity.id

Testing:
- Apply an account CR and wait until the account is ready
- Apply an accountclaim CR
- Check Amazon console for the new OU and check that the account is inside

cc @fahlmant @jharrington22

Edit: description, testing
